api/environments.md

```diff
@@ -9,8 +9,12 @@ Tradecloud has 3 environments available for customers:
 * [Production live environment](environments.md#production-environment)
 * [Acceptance test environment](environments.md#acceptance-test-environment)
 
+TODO: point to swagger docs
+
 ## Production environment
 
+TODO: this points to the api listing. Should this point to the gitbook instead?
```

Comment:
It can only point to a listing in Gitbook. The current listings will be internal behind VPN.